### DISCUSSION TITLE
Real use case for documents

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#11484] [Archive]
Any idea to manage documents?

A real use case could be the following:

1. Allow users to create each one a private document folder with already set sub-folders (Texts, Images, Videos, etc.)
2. When adding a document, ask for Type, Title, Author, Date in the intent to save that in a Collection
3. Then, Drag and drop the document, place it in the right folder (according to its Type) and save the link in the above Collection.

This would allow further search in the mentioned Collection and find easily any document by Type, Title, Author and Date.

**Comment by rijkvanzanten** [comment]
I'm not quite sure I understand the question.. Are you asking about another way to manage files in directus?

**Reply by fitu86** [reply]
Is there a way to add content to the documentation module from the app?
